misk/src/main/kotlin/misk/hibernate/ForceUTCTimeZoneService.kt

```diff
@@ -0,0 +1,19 @@
+package misk.hibernate
+
+import com.google.common.util.concurrent.AbstractIdleService
+import java.time.ZoneOffset
+import java.util.TimeZone
+
+/**
+ * This works around a Hibernate issue when loading LocalDate instances from the database when
+ * running on a machine which doesn't have UTC as the system timezone. All production servers have
+ * UTC as the system timezone so this is essentially a no op in those case.
+ */
+class ForceUTCTimeZoneService : AbstractIdleService() {
```

Comment:
awesome. I think this probably belongs somewhere other than misk.hibernate (not urgent). Also we do `ForceUtcTimeZoneService` not `ForceUTCTimeZoneService` per https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Reply:
It might belong in misk.time but currently this is only installed in the HibernateModule and strictly only necessary there (as Hibernate doesn't handle if mysqld is in a different timezone to the server).